### PR TITLE
Pin rainbow and nokogiri gems to known good versions

### PR DESCRIPTION
**Pull request by dazuma**

### CODE REVIEW

**Comment by quartzmo** [comment]
I have pulled this branch locally, running bundle update now...

File: google-cloud-error_reporting/Gemfile [hunk 1]
```diff
@@ -7,3 +7,7 @@ gem "google-cloud-core", path: "../google-cloud-core"
 gem "gcloud-jsondoc",
     git: "https://github.com/GoogleCloudPlatform/google-cloud-ruby.git",
     branch: "gcloud-jsondoc"
+
+# TEMP: rainbow (a dependency of rubocop) version 2.2 seems to have a problem,
+# so pinning to 2.1 for now.
+gem "rainbow", "~> 2.1.0"
```
Comment (blowmage) [inline google-cloud-error_reporting/Gemfile]: Error Reporting also has a dependency on nokogiri

File: google-cloud-logging/Gemfile [hunk 2]
```diff
@@ -9,3 +9,7 @@ gem "stackdriver-core", path: "../stackdriver-core"
 gem "gcloud-jsondoc",
     git: "https://github.com/GoogleCloudPlatform/google-cloud-ruby.git",
     branch: "gcloud-jsondoc"
+
+# TEMP: rainbow (a dependency of rubocop) version 2.2 seems to have a problem,
+# so pinning to 2.1 for now.
+gem "rainbow", "~> 2.1.0"
```
Comment (blowmage) [inline google-cloud-logging/Gemfile]: Stackdriver Logging also has a dependency on nokogiri

File: stackdriver/Gemfile [hunk 3]
```diff
@@ -13,3 +13,7 @@ gem "stackdriver-core", path: "../stackdriver-core"
 gem "gcloud-jsondoc",
     git: "https://github.com/GoogleCloudPlatform/google-cloud-ruby.git",
     branch: "gcloud-jsondoc"
+
+# TEMP: rainbow (a dependency of rubocop) version 2.2 seems to have a problem,
+# so pinning to 2.1 for now.
+gem "rainbow", "~> 2.1.0"
```
Comment (blowmage) [inline stackdriver/Gemfile]: Stackdriver also has a dependency on nokogiri

**Comment by dazuma** [comment]
Thanks Mike. Forgot to test with ruby 2.0. Fixed the above comments, as well as google-cloud-trace, and retested `rake bundleupdate` on ruby 2.0 and ruby 2.3.3.

**Comment by quartzmo** [comment]
Same, works for me locally.

**Comment by dazuma** [comment]
Thanks guys.

**Pull request merged**